src/mon/PGMap.cc

```diff
@@ -1621,6 +1621,16 @@ void PGMap::dump_osd_stats(ceph::Formatter *f, bool with_net) const
     f->close_section();
   }
   f->close_section();
+
+  f->open_array_section("pool_statfs");
```

Comment:
nit, how about s/pool_statfs/pool_stats/?

Reply:
there's already a "pool_stat".  "pool_statfs" matches the internal naming, and it's really per-pool 'statfs()' (objectstore stats).